docs/outlook/mobile-log-appointments.md

```diff
@@ -0,0 +1,266 @@
+                    <bt:Image size="48" scale="2" resid="icon-48"/>
+                    <bt:Image size="48" scale="3" resid="icon-48"/>
+                  </Icon>
+                  <Action xsi:type="ShowTaskpane">
+                    <SourceLocation resid="residTaskPaneUrl" />
+                  </Action>
+                </Control>
+              </Group>
+            </OfficeTab>
+          </ExtensionPoint>
+        </MobileFormFactor>
+      </Host>
+    </Hosts>
+    <Resources>
+      <bt:Images>
+        <bt:Image id="icon-16" DefaultValue="https://contoso.com/assets/icon-16.png"/>
+        <bt:Image id="icon-32" DefaultValue="https://contoso.com/assets/icon-32.png"/>
+        <bt:Image id="icon-48" DefaultValue="https://contoso.com/assets/icon-48.png"/>
+        <bt:Image id="icon-64" DefaultValue="https://contoso.com/assets/icon-64.png"/>
+        <bt:Image id="icon-80" DefaultValue="https://contoso.com/assets/icon-80.png"/>
+      </bt:Images>
+      <bt:Urls>
+        <bt:Url id="residFunctionFile" DefaultValue="https://contoso.com/commands.html"/>
+        <bt:Url id="residTaskPaneUrl" DefaultValue="https://contoso.com/taskpane.html"/>
+      </bt:Urls>
+      <bt:ShortStrings>
+        <bt:String id="residDescription" DefaultValue="Log appointment details and notes to Contoso CRM"/>
+        <bt:String id="residLabel" DefaultValue="Log to Contoso CRM"/>
+      </bt:ShortStrings>
+      <bt:LongStrings>
+        <bt:String id="residTooltip" DefaultValue="Log details and notes to Contoso CRM for this appointment."/>
+      </bt:LongStrings>
+    </Resources>
+  </VersionOverrides>
+</VersionOverrides>
+```
+
+> [!TIP]
+> To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md) and [Add support for add-in commands for Outlook Mobile](add-mobile-support.md).
+
+## Implement capturing appointment details using a UI-less command
+
+In this section, learn how your add-in can extract appointment details when the user taps the **Log** button.
+
+1. From the same quick start project, open the file **./src/commands/commands.js** in your code editor.
+
+1. Replace the entire content of the **commands.js** file with the following JavaScript.
+
+    ```js
+    // Office is ready.
+    Office.onReady(function () {
+        // Add any initialization code here.
+      }
+    );
+
+    function logCRMEvent(event) {
+      console.log(`Subject: ${Office.context.mailbox.item.subject}`);
+      Office.context.mailbox.item.body.getAsync(
+        "html",
+        { asyncContext: "This is passed to the callback" },
+        function callback(result) {
+          if (result.status === Office.AsyncResultStatus.Succeeded) {
+            event.completed({ allowEvent: true });
+          } else {
+            console.error("Failed to get body.");
+            event.completed({ allowEvent: false });
+          }
+        }
+      );
+    }
+
+    function getGlobal() {
```

Comment:
this function may not be required
-Manbendra

Reply:
I believe this is to support UI-less functionality in OWA - which should apply for the AppointmentAttendeeCommandSurface